src/cljs/rems/accept_invitation.cljs

```diff
@@ -0,0 +1,56 @@
+(ns rems.accept-invitation
+  "Page for accepting invitations v2. See `rems.actions.accept-invitation` for the v1."
+  (:require [re-frame.core :as rf]
+            [rems.atoms :refer [document-title]]
+            [rems.flash-message :as flash-message]
+            [rems.spinner :as spinner]
+            [rems.text :refer [text text-format]]
+            [rems.util :refer [navigate! post!]]))
+
+(rf/reg-event-fx
+ ::enter-page
+ (fn [{:keys [db]} [_ type token]]
+   {:db (assoc db ::type type ::token token)
+    ::accept-invitation [type token]}))
+
+(rf/reg-sub ::type (fn [db] (::type db "")))
+(rf/reg-sub ::token (fn [db] (::token db "")))
```

Comment:
do we need `type` and `token` in client db? it looks like they are only used in the `::accept-invitation` effect which gets them via event effect

Reply:
This is leftover from the other invitation page that I used as an example. Since we don't need them I'll just remove them. :+1: